src/SettingStore.php

```diff
@@ -48,6 +48,29 @@ public function get($key, $default = null)
 	}
 
 	/**
+	 * Get a flattened array from the settings data.
+	 *
+	 * @param  array   $array   Optional array of settings values, with a fallback to all settings.
+	 * @param  string  $prefix  Optional prefix.
```

Comment:
I would expand the description a bit. "Optional prefix for keys in the returned array" or something like that.